chromium_src/ios/web/webui/url_data_source_ios.mm

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "ios/web/public/webui/url_data_source_ios.h"
+
+#include "base/containers/span.h"
+#include "base/no_destructor.h"
+#include "base/strings/strcat.h"
+#include "base/strings/stringprintf.h"
+#include "base/strings/utf_string_conversions.h"
+#include "ios/chrome/browser/shared/model/url/chrome_url_constants.h"
```

Comment:
What is this include for? I don't see any usage of `kChromeUI` prefixed variables anywhere in this override? Was `kChromeUIUntrustedScheme` previously defined here? Seems like this should be removed to resolve possible layering violations + a missing BUILD.gn to include a dep that doesn't currently exist in `//ios/web/webui`